github/ci/prow-deploy/files/jobs/kubevirt/bridge-marker/bridge-marker-presubmits-0.9.1.yaml

```diff
@@ -0,0 +1,33 @@
+---
+presubmits:
+  kubevirt/bridge-marker:
+    - name: pull-bridge-marker-test-0.9.1
+      branches:
+        - release-0.9.1
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      cluster: prow-workloads
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20220512-78048f1
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
```

Comment:
From what I can see - this job uses the default values for bringing up the 1.23 k8s provider from kubevirtci. I think this job could use a memory request amount similar to the check-provision jobs (14Gi) - this could help with getting the prow job scheduled quicker?

Reply:
I see no reason not to, let me update the PR